src/types/nft.ts

```diff
@@ -47,3 +51,11 @@ export type NFTMetadataResponse = {
     links: Record<NFTMetadataLinksProviders, string>;
   } | null;
 };
+
+export type NFTCollectionMetadataResponse = {
+  status: 200 | 404 | 500;
```

Comment:
Just wondering, the status can never be something else?...
(same for NFTMetadataResponse above)

Reply:
I don't think so no, and it's not in the Swagger so I'll ask Rémi for more precision 👌